### DISCUSSION TITLE
Auto Level Recalc

### DISCUSSION THREAD

**Discussion by exzolink** [BG-Software-LLC/SuperiorSkyblock2#659] [Pending Feature Requests]
Sorry, didnt notice in config.

If its so laggy, maybe implement auto recalc on quit or join event and only for that player?

**Comment by OmerBenGera** [comment]
I already answered this.
There is no need to open new feature requests.
Levels are calculated when you place and break blocks, therefore its unnecessary.
If you really want something like this, you can make an addon to the plugin that does it.
